docs/semgrep-secrets/getting-started.md

```diff
@@ -46,6 +46,85 @@ After you've enabled Semgrep Secrets, you can:
 * Wait for your scheduled Semgrep full scan
 * Open a pull request or merge request and wait for Semgrep to scan the branch automatically
 
+## View your findings
+
+You can use Semgrep AppSec Platform's **Secrets** page to view the findings generated by Semgrep Secrets after it scans your codebase. To access the [**Secrets** page](https://semgrep.dev/orgs/-/secrets):
+
+1. Log in to [Semgrep AppSec Platform](https://semgrep.dev/login).
+2. Click **[Secrets](https://semgrep.dev/orgs/-/secrets)** to navigate to your results.
+
+### Secrets page structure
+
+The **Secrets** page consists of:
+
+- The **filter panel**, which you can use to group and filter for specific findings
+- Information about findings identified by Semgrep Secrets. Each finding in the list includes:
+  - When the finding was created
+  - The type of secret found and where in the code it is located, including its Project and branch information
+  - Its severity level
+  - Whether it has been validated by a Semgrep validator
+  - Whether it is a historical finding
+  - For users of the Semgrep Jira integration: whether there is a Jira ticket that accompanies that finding
+
+### Filter findings
+
+Semgrep AppSec Platform offers a variety of filters to help you narrow down the
+list of findings. The following sections describe what filters are available to
+you.
+
+#### Triage status
+
+| Status | Description |
+| -----------  | ------------ |
+| **Open** | Findings are open by default. A finding is open if it was present the last time Semgrep scanned the code and it has not been ignored. An open finding represents a match between the code and a rule that is enabled in the repository. Open findings require action, such as rewriting the code to eliminate the detected vulnerability. |
```

Comment:
```suggestion
| **Open** | Findings are open by default. A finding is open if it was present the last time Semgrep scanned the code and hasn't been triaged. An open finding represents a match between the code and a rule that is enabled in the repository. Open findings require action, such as rewriting the code to eliminate the detected vulnerability. |
```
(Because we now have many triage states aside from Ignored)

Reply:
I went and checked - secrets still doesn't have them even with "latest" jira. i'll close this comment